### PR TITLE
BZ1775456: Add healthz/ready for ingress LB

### PR DESCRIPTION
**Pull request by xenolinux**
Fixes: 1775456
OCP version: 4.8, 4.9, 4.10
[Preview](https://deploy-preview-39833--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere.html#installation-load-balancing-user-infra_installing-vsphere)
QA contact: @jinyunma

### CODE REVIEW

**Review by jeana-redhat — APPROVED** [review]

**Comment by jinyunma** [comment]
@lihongan could you please help to review this doc PR? Does route on vsphere upi cluster already cover this? Or need to add additional configurations in external LB server for ingress LB?

File: modules/installation-load-balancing-user-infra.adoc [hunk 1]
```diff
@@ -142,6 +142,12 @@ Configure the following ports on both the front and back of the load balancers:
 |X
 |HTTP traffic
 
+|`1936`
+|The worker nodes that run the Ingress Controller pods, by default. You must configure the `/healthz/ready` endpoint for the API server health check probe.
```
Comment (lihongan) [inline modules/installation-load-balancing-user-infra.adoc:146]: It is not for the `... API server health check probe` but `ingress health check probe`.

Comment (xenolinux) [reply modules/installation-load-balancing-user-infra.adoc]: acked

**Comment by lihongan** [comment]
LGTM, thank you!

**Comment by xenolinux** [comment]
@jeana-redhat this is ready to merge. Thanks in advance!

**Pull request merged**